scene/resources/style_box.h

```diff
@@ -230,11 +230,11 @@ class StyleBoxFlat : public StyleBox {
 // just used to draw lines.
 class StyleBoxLine : public StyleBox {
 	GDCLASS(StyleBoxLine, StyleBox);
-	Color color;
-	int thickness;
-	bool vertical;
-	float grow_begin;
-	float grow_end;
+	Color color = Color(0.0, 0.0, 0.0);
```

Comment:
```suggestion
	Color color;
```
That's the default color (`0, 0, 0, 1`)